rs/embedders/benches/compilation.rs

```diff
@@ -78,12 +79,16 @@ fn generate_binaries() -> Vec<(String, BinaryEncodedWasm)> {
         BinaryEncodedWasm::new(wat::parse_str(many_funcs).expect("Failed to convert wat to wasm")),
     ));
 
-    // TODO(EXC-1985): Modify the open chat canister so that it runs.
-    // // This benchmark uses a real-world wasm which is stored as a binary file in this repo.
-    // let real_world_wasm =
-    //     BinaryEncodedWasm::new(include_bytes!("test-data/user_canister_impl.wasm").to_vec());
+    // This benchmark uses the open chat user canister which is stored as a
+    // binary file in this repo.  It is generated from
+    // https://github.com/adambratschikaye/open-chat/tree/abk/for-replica-benchmarking
```

Comment:
Should we maybe move this to a repository under dfinity? I'm thinking that we should make it something that is owned by the team not on a personal repo.

Reply:
I need to get some more permissions enabled first.

Reply:
Fixed now.